docs/src/pages/customization/transitions/transitions.md

```diff
@@ -0,0 +1,33 @@
+# Transitions
+
+<p class="description">Use the <code>theme.transitions.create()</code> helper to create consistent transitions on the elements of your UI.</p>
```

Comment:
the generated HTML is:

```html
<meta name="description" content="Use the &lt;code&gt;theme.transitions.create()&lt;/code&gt; helper to create consistent transitions on the elements of your UI."/>
```

I'm curious to see what it looks like once indexed. We will see :)